plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -101,9 +102,14 @@ function perflab_query_plugin_info( string $plugin_slug ) {
 		}
 	}
 
+	if ( ! isset( $plugins[ $plugin_slug ] ) ) {
+		// Cache the fact that the plugin was not found.
+		$plugins[ $plugin_slug ] = false;
+	}
+
 	set_transient( $transient_key, $plugins, HOUR_IN_SECONDS );
```

Comment:
Here I think it would be good to check if there was an error condition, and if so, set the expiration to 1 minute instead of `HOUR_IN_SECONDS`.